playbooks/provisioning/openstack/galaxy-requirements.yaml

```diff
@@ -0,0 +1,5 @@
+# This is the Ansible Galaxy requirements file to pull in the correct roles
+
+# From 'infra-ansible'
+src: https://github.com/redhat-cop/infra-ansible
```

Comment:
This must be in a YAML list, otherwise it fails. Like so:

```yaml
---
# This is the Ansible Galaxy requirements file to pull in the correct roles

# From 'infra-ansible'
- src: https://github.com/redhat-cop/infra-ansible
  version: master
```

(and please also add the `---` on top so the travis stops complaining)